Competition2017/src/competition/subsystems/servo/ServoSubsystem.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.servo;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XServo;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class ServoSubsystem extends BaseSubsystem {
```

Comment:
I think we should specify what servos are being controlled. In this case, I'd recommend `ShooterAimAdjustmentSubsystem` and rename the packages and variables accordingly. You can choose though; the idea is that these servos aim the shooter pads a bit to extend the arc.